server/src/main/java/org/apache/iotdb/db/engine/cache/TsFileMetaDataCache.java

```diff
@@ -55,24 +54,24 @@
   /**
    * estimated size of a deviceMetaDataMap entry in TsFileMetaData.
```

Comment:
estimated size of metadataIndex entry in TsFileMetaData.

Reply:
Thanks for this good catch!